packages/block-library/src/page-list/edit.js

```diff
@@ -109,7 +109,11 @@ function BlockContent( {
 	}
 
 	if ( pages.length > 0 ) {
-		return <ul { ...innerBlocksProps }></ul>;
+		return (
+			<nav>
+				<ul { ...innerBlocksProps }></ul>
+			</nav>
+		);
```

Comment:
I would recommend excluding this block from the PR since the page list can be used in the navigation block. Adding the nav here invalidates the HTML and may break the navigation block in unexpected ways.